.travis.yml

```diff
@@ -99,12 +99,15 @@ script:
         cd "$BUILD_IN_DIR"
         cmake $CMAKE_ARGS "$SOURCE_DIRECTORY"
     fi
-  - make && sudo env PATH=$PATH make install && awesome --version && make check
-  - |
-    if [ "$DO_COVERAGE" = "true" ]; then
-        make check-coverage || exit 1;
+  - make && sudo env PATH=$PATH make install && awesome --version
+  - if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
```

Comment:
You might have to keep the `- |`-line if you want multi-line stuff to work. At least the travis failure sounds like it.